docs/developer/release-process.md

```diff
@@ -1,58 +1,54 @@
 # How to make a release
 
-After ensuring the release is done and integration tests work:
+These instructions apply to the current master, some branches might have older scripts and require more manual work. Please read the instructions from that branch and do your diligence.
+
+After ensuring the work for the release is done and integration tests work:
 
 Run the following sequence, doing the small manual steps when required (until automated). There is no need to make a release branch as we want to maintain documentation in the master also.
 Ensure the post-release and pre-release scripts are correct if you've made any changes to the structure of the project since the last release.
 
-The steps required (assuming releasing v1.10.0 and previous one was v1.9.0), doing from master (replace git commands with correct target remote names):
+The steps required (assuming releasing v1.13.0), doing from master (replace git commands with correct target remote names):
```

Comment:
Where are the remote names listed?

Reply:
It depends what you've set on your computer. You can do ``git remote -v`` to see them. I use ``upstream`` and ``origin``, but you can have different names.